### ISSUE TITLE
[Error] Sidekiq Redis configuration

### ISSUE THREAD

**Issue by shaman007** [Freika/dawarich#376]
**Describe the bug**

Some sidekiq jobs gon't get Redis configuraion. At some parts of Sidekiq there is localhost still, not an expternal configured Redis.

**Version**
tag:latest

**To Reproduce**
1. configure external Redis, so

```
2024-11-08T14:41:58.915Z pid=85 tid=3x9 INFO: Sidekiq 7.3.4 connecting to Redis with options {:size=>10, :pool_name=>"internal", :url=>"redis://:REDACTED@redis-master.redis.svc.cluster.local:6379/10"}
2024-11-08T14:41:58.944Z pid=85 tid=3x9 INFO: Sidekiq 7.3.4 connecting to Redis with options {:size=>10, :pool_name=>"default", :url=>"redis://:REDACTED@redis-master.redis.svc.cluster.local:6379/10"}
```
3. See errors:
```
D, [2024-11-08T14:41:59.664197 #85] DEBUG -- : ↳ app/services/visits/suggest.rb:80:in `create_visits_notification'
E, [2024-11-08T14:41:59.706237 #85] ERROR -- : Error performing VisitSuggestingJob (Job ID: 501e5681-ad76-47c9-bacb-eadde4f0fe11) from Sidekiq(visit_suggesting) in 414.37ms: Redis::CannotConnectError (Connection refused - connect(2) for 127.0.0.1:6379 (redis://localhost:6379/1)):
/usr/local/lib/ruby/3.3.0/socket.rb:1219:in `__connect_nonblock'
/usr/local/lib/ruby/3.3.0/socket.rb:1219:in `connect_nonblock'
/usr/local/lib/ruby/3.3.0/socket.rb:60:in `connect_internal'
/usr/local/lib/ruby/3.3.0/socket.rb:141:in `connect'
/usr/local/lib/ruby/3.3.0/socket.rb:647:in `block in tcp'
/usr/local/lib/ruby/3.3.0/socket.rb:231:in `each'
/usr/local/lib/ruby/3.3.0/socket.rb:231:in `foreach'
/usr/local/lib/ruby/3.3.0/socket.rb:637:in `tcp'
```

**Comment by hernil** [comment]
I see the same in my logs. Not sure what it means as I've only played around with this application for a few hours trying to get my data imported. 
My Sidekiq dashboard reports 
> 115,591 Processed
> 115,591 Failed 

So that's a bummer  but there's a bunch enqueued as well.

**Comment by Freika** [comment]
Fixed in 0.16.3
